core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -0,0 +1,339 @@
+# coding: utf-8
```

Comment:
Hey, if I look at the `update_exploration` function that was used previously for saving the exploration after migration there are some lines that will probably also need to be handled in this job https://github.com/oppia/oppia/blob/d27760036635d77e322c8d63d994d19a41897391/core/domain/exp_services.py#L1145-L1168

From my quick look:
- lines 1145 to 1146 might be needed to be implemented
- line 1150 might be needed to be implemented
- lines 1153 to 1156 are already implemented in your job
- lines 1158 to 1163 are not relevant to migrations
- lines 1165 to 1168 might be needed to be implemented

Reply:
Done.